hybrid/HybridSampleApps/NoteSync/src/com/salesforce/samples/notesync/ContentSoqlSyncDownTarget.java

```diff
@@ -141,11 +145,9 @@ private RestRequest buildQueryMoreRequest(String sessionId, String locator) thro
     private RestRequest buildSoapRequest(String sessionId, String body) throws UnsupportedEncodingException {
         Map<String, String> customHeaders = new HashMap<String, String>();
         customHeaders.put("SOAPAction", "\"\"");
+        RequestBody requestBody = RequestBody.create(MEDIA_TYPE_XML, String.format(REQUEST_TEMPLATE, sessionId, body));
 
-        StringEntity entity = new StringEntity(String.format(REQUEST_TEMPLATE, sessionId, body), HTTP.UTF_8);
-        entity.setContentType("text/xml");
-
-        return new RestRequest(RestRequest.RestMethod.POST, "/services/Soap/u/36.0", entity, customHeaders);
+        return new RestRequest(RestRequest.RestMethod.POST, "/services/Soap/u/36.0", requestBody, customHeaders);
```

Comment:
We could do that, soap however wants just the number (e.g. "36.0") while rest needs "v36.0".